minimal_test_cases/Assets/force_torque_issue/AddForce.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AddForce : MonoBehaviour
+{
+    public float forceStrength;
+    public float RayDistance;
+
+    // empty child object of Magnet - used to set the raycast begin point in world
+    private Transform _tipTransform;
+    
+    void Start()
+    {
+        // set force strength
+        if (forceStrength <= 0)
+        {
+            forceStrength = 5;
+            Debug.Log("Set defualt force: " + forceStrength);
+        }
```

Comment:
Having defaults isn't really adding much towards conveying the problem at hand, and just results in additional code & logic that readers need to parse through.  I'd suggesting stripping this code out, along with the other defaults.
As an alternative, it's also reasonable to have error checking on the input values, and then print errors (or crash the app) if the values are not set as expected.